examples/simple-passthrough-to-confluent-cloud/docker-compose.yaml

```diff
@@ -0,0 +1,20 @@
+services:
+  gateway:
+    image: "${GATEWAY_IMAGE:-confluentinc/cpc-gateway:latest}"
+    container_name: cc-passthrough-gateway
+    volumes:
+      - ./gateway.yaml:/etc/gateway/gateway.yaml:ro
+      - ./ssl:/etc/gateway/ssl:ro
+    environment:
+      GATEWAY_CONFIG_FILE: /etc/gateway/gateway.yaml
+    ports:
+      - "19092:19092"
+      - "19093:19093"
+      - "19094:19094"
+      - "19095:19095"
+      - "9190:9190"
```

Comment:
we should open more ports. I guess 13 ports (1 bootstrap + 12 brokers) for standard cluster.